docs/copilot/copilot-extensibility-overview.md

```diff
@@ -10,34 +10,46 @@ MetaSocialImage: images/shared/github-copilot-social.png
 ---
 # GitHub Copilot extensibility in VS Code
 
-GitHub Copilot is an AI coding assistant that helps you write code faster and with less effort. You can get code suggestions as you type in the editor, chat with Copilot to ask for help with your code, or get help with writing tests, fixing bugs, or refactoring code.
+You can extend GitHub Copilot in Visual Studio Code to expand the functionality of Copilot Chat or to take advantage of Copilot's Language Learning Model (LLM) to build AI features into your extension.
```

Comment:
"Language Learning Model" should be "Large Language Models"

Reply:
I used the terminology GitHub uses. Which one should be used? https://docs.github.com/en/copilot/building-copilot-extensions/building-a-copilot-agent-for-your-copilot-extension/using-copilots-llm-for-your-agent